src/main/java/ch/njol/skript/expressions/ExprSeaLevel.java

```diff
@@ -0,0 +1,55 @@
+/**
+ *   This file is part of Skript.
+ *
+ *  Skript is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  Skript is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with Skript.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ *
+ * Copyright 2011-2017 Peter Güttinger and contributors
+ */
+package ch.njol.skript.expressions;
+
+import org.bukkit.World;
+
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.expressions.base.SimplePropertyExpression;
+
+@Name("Sea Level")
+@Description("Gets the sea level of a world.")
+@Examples("send \"The sea level in your world is %sea level in player's world%\"")
+@Since("INSERT VERSION")
+public class ExprSeaLevel extends SimplePropertyExpression<World, Number> {
+	
+	static {
+		register(ExprSeaLevel.class, Number.class, "sea level", "worlds");
+	}
+	
+	@Override
+	public Number convert(World world) {
+		return world.getSeaLevel();
+	}
+	
+	@Override
+	public Class<? extends Long> getReturnType() {
```

Comment:
Oh, this should extend and return Number too.

Reply:
Oopsie, didn't see that 😅. Should be fixed now!